solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -547,7 +552,12 @@ private synchronized void initializeAuthenticationPlugin(
 
     // Initialize the plugin
     if (pluginClassName != null) {
-      log.info("Initializing authentication plugin: {}", pluginClassName);
+      if (authPluginList.contains(pluginClassName)) {
+        log.info("Initializing authentication plugin: {}", pluginClassName);
+      } else {
+        log.warn("Wrong Authentication plugin {} configured." +
```

Comment:
I wonder if this needs to be more of a "The plugin X doesn't match any of the default plugins shipped with Solr"...   Wouldn't there be a possibility of someone writing their own plugin, so it wouldn't be in the authPluginList?

Reply:
Yes Eric, I am aiming this PR just to validate solr's inbuilt Auth plugins write WARN logs if it is not matching as per Solr 9+ documentation. It will help users to, easily, find what mistake they commited.